cfme/web_ui/__init__.py

```diff
@@ -3349,7 +3349,7 @@ def select_by_value(self, value):
         index = options_map.index(value)
         if not self.is_open:
             self.open()
-        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]'.format(index)
+        new_loc = self._loc + '/../div/ul/li[@data-original-index={}]/a'.format(index)
```

Comment:
the select_by_values method is used in adding buttons group (Automate->Customization), 
and without /a we couldn't select the button image from drop down list.